packages/twenty-server/src/engine/core-modules/auth/services/auth.service.ts

```diff
@@ -474,12 +478,12 @@ export class AuthService {
       locale: firstUserWorkspace.locale,
     });
 
-    const html = await render(emailTemplate, { pretty: true });
-    const text = await render(emailTemplate, { plainText: true });
+    const html = render(emailTemplate, { pretty: true });
```

Comment:
Good catch, I see many other await render in the codebase, would you mind also cleaning them up?